test/k8sT/Bandwidth.go

```diff
@@ -30,8 +30,12 @@ var _ = Describe("K8sBandwidthTest", func() {
 	})
 
 	AfterAll(func() {
-		_ = kubectl.Delete(demoYAML)
+		kubectl.Delete(demoYAML)
 		ExpectAllPodsTerminated(kubectl)
+
+		UninstallCiliumFromManifest(kubectl, ciliumFilename)
+		ExpectAllPodsTerminated(kubectl)
```

Comment:
Nice catch!

Why are we only seeing this now? Should we backport this fix?

Reply:
I couldn't figure out which K8s change might have caused this :man_shrugging: 

Anyway, good idea re backporting. I will do it after it gets merged.